.github/workflows/build-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Build image and publish to Docker Hub
+
+on:
+  push:
+    tags:
+      - 'v-[0-9]{4}-[0-9]{2}-[0-9]{1,2}.[0-9]+'
```

Comment:
I would think it should be `\.` instead of `.` but the [semver example](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags) in the docs says otherwise,

Reply:
it looks like they're patterns, not regexes, so the guidelines are probably correct.